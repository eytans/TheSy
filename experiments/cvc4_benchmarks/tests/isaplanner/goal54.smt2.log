19:20:22 [WARN] Collecting statistics
19:20:22 [WARN]  ex is: zero
19:20:22 [WARN]  ex is: (succ zero)
19:20:22 [WARN]  ex is: (succ (succ zero))
19:20:22 [WARN] Running exploration without proof mode, but goals were given
19:20:22 [WARN] Proof mode: true
19:20:22 [WARN]  ex is: zero
19:20:22 [WARN]  ex is: (succ zero)
19:20:22 [WARN]  ex is: (succ (succ zero))
19:20:22 [WARN] Running exploration without proof mode, but goals were given
19:20:22 [WARN] Proof mode: true
19:20:22 [WARN] proved: (minus ?ts_ph_Nat_0 ?ts_ph_Nat_0) => zero
19:20:22 [WARN] proved: (plus ?ts_ph_Nat_0 zero) => ?ts_ph_Nat_0
19:20:22 [WARN] proved: (leq zero ?ts_ph_Nat_0) => true
19:21:13 [WARN] proved: (plus ?ts_ph_Nat_0 (succ ?ts_ph_Nat_1)) => (succ (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1))
19:21:13 [WARN] proved: (succ (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1)) => (plus ?ts_ph_Nat_0 (succ ?ts_ph_Nat_1))
19:21:13 [WARN] Found all lemmas
