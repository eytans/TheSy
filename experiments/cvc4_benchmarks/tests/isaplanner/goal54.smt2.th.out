19:20:22 [0m[33m[WARN] [0mCollecting statistics
(leq ?__x0 ?__y1) => (or (= ?__x0 ?__y1) (less ?__x0 ?__y1))
(or (= ?__x0 ?__y1) (less ?__x0 ?__y1)) => (leq ?__x0 ?__y1)
(plus zero ?n) => ?n
(plus (succ ?n) ?m) => (succ (plus ?n ?m))
(succ (plus ?n ?m)) => (plus (succ ?n) ?m)
(minus zero ?n) => zero
(minus ?n zero) => ?n
(minus (succ ?n) (succ ?m)) => (minus ?n ?m)
(minus ?n ?m) => (minus (succ ?n) (succ ?m))
Searcher: (minus ?autovar4 (succ ?autovar2))
Searcher: (minus (succ ?autovar3) ?autovar5)
19:20:22 [0m[33m[WARN] [0m ex is: zero
19:20:22 [0m[33m[WARN] [0m ex is: (succ zero)
19:20:22 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:20:22 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:20:22 [0m[33m[WARN] [0mProof mode: true
19:20:22 [0m[33m[WARN] [0m ex is: zero
19:20:22 [0m[33m[WARN] [0m ex is: (succ zero)
19:20:22 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:20:22 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:20:22 [0m[33m[WARN] [0mProof mode: true
19:20:22 [0m[33m[WARN] [0mproved: (minus ?ts_ph_Nat_0 ?ts_ph_Nat_0) => zero
19:20:22 [0m[33m[WARN] [0mproved: (plus ?ts_ph_Nat_0 zero) => ?ts_ph_Nat_0
19:20:22 [0m[33m[WARN] [0mproved: (leq zero ?ts_ph_Nat_0) => true
19:21:13 [0m[33m[WARN] [0mproved: (plus ?ts_ph_Nat_0 (succ ?ts_ph_Nat_1)) => (succ (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1))
19:21:13 [0m[33m[WARN] [0mproved: (succ (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1)) => (plus ?ts_ph_Nat_0 (succ ?ts_ph_Nat_1))
19:21:13 [0m[33m[WARN] [0mFound all lemmas
done in 50739
